src/components/pages/divisionsPage/divisionsPage.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+import {
+  Box,
+  Heading,
+  HorizontalRule,
+} from "@nypl/design-system-react-components";
+import PageLayout from "../../pageLayout/pageLayout";
+import { headerBreakpoints } from "src/utils/breakpoints";
+import SwimLanes from "src/components/swimlanes/swimLanes";
+import { props } from "__tests__/data/swimlaneProps";
+import useBreakpoints from "src/hooks/useBreakpoints";
+
+export default function DivisionsPage() {
+  const { isLargerThanLargeTablet, isLargerThanLargeMobile } = useBreakpoints();
+  const numColumns = isLargerThanLargeTablet
+    ? 4
+    : isLargerThanLargeMobile
+    ? 2
+    : 1;
```

Comment:
This is the same as what's in `HomePageMainContent`. I think now is a good time to make this into a custom hook to reuse it in both components, and maybe any other ones in the future.